diff --git a/node_modules/@langchain/core/dist/messages/ai.js b/node_modules/@langchain/core/dist/messages/ai.js
index d881876..908fed8 100644
--- a/node_modules/@langchain/core/dist/messages/ai.js
+++ b/node_modules/@langchain/core/dist/messages/ai.js
@@ -29,12 +29,6 @@ var AIMessage = class extends BaseMessage {
 			initParams = fields;
 			const rawToolCalls = initParams.additional_kwargs?.tool_calls;
 			const toolCalls = initParams.tool_calls;
-			if (!(rawToolCalls == null) && rawToolCalls.length > 0 && (toolCalls === void 0 || toolCalls.length === 0)) console.warn([
-				"New LangChain packages are available that more efficiently handle",
-				"tool calling.\n\nPlease upgrade your packages to versions that set",
-				"message tool calls. e.g., `pnpm install @langchain/anthropic`,",
-				"pnpm install @langchain/openai`, etc."
-			].join(" "));
 			try {
 				if (!(rawToolCalls == null) && toolCalls === void 0) {
 					const [toolCalls$1, invalidToolCalls] = defaultToolCallParser(rawToolCalls);